examples/send_ckb_example.rs

```diff
@@ -0,0 +1,53 @@
+use ckb_sdk::{
+    transaction::{
+        builder::{CkbTransactionBuilder, SimpleTransactionBuilder},
+        input::InputIterator,
+        signer::{SignContexts, TransactionSigner},
+        TransactionBuilderConfiguration,
+    },
+    Address, CkbRpcClient, NetworkInfo,
+};
+use ckb_types::{core::Capacity, h256};
+use std::{error::Error as StdErr, str::FromStr};
+
+fn main() -> Result<(), Box<dyn StdErr>> {
+    let network_info = NetworkInfo::testnet();
+    let sender = "ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsq2qf8keemy2p5uu0g0gn8cd4ju23s5269qk8rg4r";
+    let receiver="ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqv5dsed9par23x4g58seaw58j3ym5ml2hs8ztche";
+
+    let configuration = TransactionBuilderConfiguration::new_with_network(network_info.clone())?;
+    // set small change action instead of default
+    // use ckb_sdk::transaction::SmallChangeAction;
+    // configuration.small_change_action = SmallChangeAction::AsFee { threshold: Capacity::bytes(61)?.as_u64() };
+    // configuration.small_change_action = SmallChangeAction::to_output(&sender.parse()?, Capacity::bytes(1)?.as_u64());
+
+    let addr = Address::from_str(sender)?;
+    let receiver = Address::from_str(receiver)?;
+    let iterator = InputIterator::new_with_address(&[addr], configuration.network_info());
+    let mut builder = SimpleTransactionBuilder::new(configuration, iterator);
+    let addr = Address::from_str(sender)?;
```

Comment:
duplicated with Line 24?

Reply:
addr moved in line 26, so clone line 26, or it can be created again here.